packages/email/src/render.ts

```diff
@@ -14,21 +13,17 @@ winston.add(console);
 export async function render(template: Generic): Promise<string>;
 export async function render(template: welcomeMessage,): Promise<string>;
 export async function render(template: Template): Promise<string> {
-  const path = `src/templates/${template.name}/template.ejs`;
-
-  // this is probably lil weird no? @niekcandaele
   try {
-    fs.access(path);
-  } catch (error) {
-    winston.error(error);
-    winston.error('The requested template does not exist.');
+    const result = await ejs.renderFile(`src/templates/${template.name}/template.ejs`, template.data, { async: true });
+    const out = mjmml2html(result);
+    if (out.errors.length > 0) {
+      winston.error(`Email template ${template.name} generated the following errors: `, out.errors);
+      return '';
+    }
+    return out.html;
   }
-
-  const result = await ejs.renderFile(path, template.data, { async: true });
-  const out = mjmml2html(result);
-  if (out.errors.length > 0) {
-    winston.error(`Email template ${template.name} generated the following errors: `, out.errors);
-    return '';
+  catch (err) {
+    winston.error(`unexpected error: ${err}`);
   }
-  return out.html;
+  return '';
 };
```

Comment:
What happens when an error happens now? An empty template is generated and will be sent to the users, no? I think it'd be better to throw the error still.